tsconfig.json

```diff
@@ -1,13 +1,13 @@
 {
   "compilerOptions": {
-    "target": "ES2019",
-    "module": "commonjs",
+    "target": "ESNext",
```

Comment:
Yes, i already changed the requirements on the package.json and updated the workflows

Reply:
I think for 1.0 I'll require Node 14.x or better since 12.x fell out of LTS. But, I'll want to double-check internally (i.e. ask @leibale what he's doing for node-redis) before I make that call.